version: '3.5'
services:
    postgres-auth:
        restart: always
        image: postgres:10
        env_file:
            - db-vars.env
        volumes:
            - auth-data:/var/lib/postgresql/data
        ports:
            - "5432:5432"
#        networks:
#            - auth_network

    auth-app:
        restart: always
        # network_mode: host
        build: .
        ports:    
            - 8000:8000
        depends_on: 
            - postgres-auth

    postgres-products:
        restart: always
        image: postgres:10
        env_file:
            - ./../flask-products/db-config.env
        volumes:
            - db-data:/var/lib/postgresql/data
        ports:
            - "5433:5432"

    products-app:
        restart: always
        build: ./../flask-products
        tty: true   # output za testovite, probaj bez 
        ports:    
            - 5000:5000
        depends_on: 
            - postgres-products
        # entrypoint: ["python", "app.py"]
#        networks: 
#            - products_network

#        networks:
#            - auth_network

#networks:
#  auth_network:
#    name: project_network

volumes:
  auth-data:
    driver: local
  db-data:
    driver: local